Octokit/Http/HttpClientAdapter.cs

```diff
@@ -238,18 +245,28 @@ internal class RedirectHandler : DelegatingHandler
         [SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope")]
         private static HttpRequestMessage CopyRequest(HttpRequestMessage oldRequest)
         {
-            var newrequest = new HttpRequestMessage(oldRequest.Method, oldRequest.RequestUri);
+            var newRequest = new HttpRequestMessage(oldRequest.Method, oldRequest.RequestUri);
 
             foreach (var header in oldRequest.Headers)
             {
-                newrequest.Headers.TryAddWithoutValidation(header.Key, header.Value);
+                newRequest.Headers.TryAddWithoutValidation(header.Key, header.Value);
             }
             foreach (var property in oldRequest.Properties)
             {
-                newrequest.Properties.Add(property);
+                newRequest.Properties.Add(property);
             }
 
-            return newrequest;
+            return newRequest;
+        }
+    }
+
+    internal class RedirectHandler : DelegatingHandler
+    {
+        public bool Enabled { get; set; }
```

Comment:
I don't see this value being used anywhere (I suspect it also wasn't used before).